AppCenterReactNativeShared/android/src/main/java/com/microsoft/appcenter/reactnative/shared/AppCenterReactNativeShared.java

```diff
@@ -58,6 +59,19 @@ public static synchronized void configureAppCenter(Application application) {
             AppCenterLog.debug(LOG_TAG, "Configure with secret.");
             AppCenter.configure(application, sAppSecret);
         }
+
+        /*
+         * When sStartAutomatically flag is set to true, every service (analytics/auth/crashes/etc.)
+         * will be started by separate AppCenter.start call. If Auth module is used,
+         * call doNotResetAuthAfterStart to avoid resetting the auth token.
+         */
+        try {
+            Class.forName("com.microsoft.appcenter.auth.Auth");
```

Comment:
What if Auth class is loaded but not registered?

Reply:
What case exactly do you mean?

Reply:
It isn't the case here

Reply:
Talked offline. This is the only case when it auto-starts so it should be fine.